java/org/contikios/cooja/Cooja.java

```diff
@@ -2668,11 +2669,12 @@ public static void loadExternalToolsDefaultSettings() {
     Properties settings = new Properties();
     try (var in = Cooja.class.getResourceAsStream(EXTERNAL_TOOLS_SETTINGS_FILENAME)) {
       if (in == null) {
-        throw new FileNotFoundException(EXTERNAL_TOOLS_SETTINGS_FILENAME + " not found");
+        throw new MissingResourceException(
+                "Could not find " + EXTERNAL_TOOLS_SETTINGS_FILENAME + ", jar file seems broken", "Cooja", "");
       }
       settings.load(in);
     } catch (IOException e) {
-      logger.warn("Error reading " + EXTERNAL_TOOLS_SETTINGS_FILENAME);
+      throw new RuntimeException(e);
```

Comment:
What do you think about throwing `MissingResourceException` also here?

Reply:
The resource isn't missing in the catch, something else went wrong if we're here.

The method is called from some menu context, so we can't put a `throws MissingResourceException` on it unless we want to catch and ignore exceptions in the caller that isn't the Cooja constructor.

Reply:
True, but it is trying and fails to read some resources. `MissingResourceException` extends `RuntimeException` so there is no need to add a throws clause. After all, this PR already adds a throw of `MissingResourceException` in the method.

Reply:
But `RuntimeException` if also fine for me. I can merge this PR as is if you like?

Reply:
Oh, I thought the method needed a throws for the change you suggested which I didn't want since it would affect other callers. I'll make the change you suggested later tonight.

Reply:
Updated according to your suggestion.